### PR TITLE
Update README with information on the annotations

### PR DESCRIPTION
**Pull request by JustinKuli**
This documents the stand-alone annotations we currently have.

Signed-off-by: Justin Kulikauskas <jkulikau@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,17 @@ kubectl -n my-managed-cluster annotate managedclusteraddon config-policy-control
 
 Any values in the
 [Helm chart's values.yaml](./pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml) can
-be modified in this way.
+be modified in with the `addon.open-cluster-management.io/values` annotation. However, the structure
```
Comment (dhaiducek) [inline README.md:81]:
```suggestion
be modified with the `addon.open-cluster-management.io/values` annotation. However, the structure
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Looks great! I'm a little embarrassed I didn't add these docs along with my PR!

Minor update requested.

**Comment by dhaiducek** [comment]
Please check DCO here.

**Comment by JustinKuli** [comment]
Sorry, I didn't realize that the `Signed-off-by` needed to be separated from the `Co-authored-by` by a blank line.

**Pull request merged**